code/modules/research/techweb/all_nodes.dm

```diff
@@ -661,6 +660,26 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
 	export_price = 5000
 
+/datum/techweb_node/med_scanning
+	id = "med_scanner"
+	tech_tier = 3
+	display_name = "Medical Scanning"
+	description = "NT didn't trust us with their design data for this, so we figured it out ourselves."
```

Comment:
Change this to something that doesn't imply the station is going against nanotrasen with their research. All research comes from nanotrasen anyway, since we are buying it off them with bitcoin (its stupid, I know).